src/views/login/index.tsx

```diff
@@ -1,3 +1,4 @@
-export const Login: React.FC<{}> = () => (
-  <h1>Login</h1>
-);
+import Login from "../../components/Login";
+export const LoginView: React.FC<{}> = () => (
```

Comment:
Leave a blank line between statement kinds.

```suggestion
import Login from "../../components/Login";

export const LoginView: React.FC<{}> = () => (
```